en/micro-integrator/docs/setup/user_stores/setting_up_ro_ldap.md

```diff
@@ -84,27 +78,87 @@ Follow the steps given below to connect the Micro Integrator to the LDAP user st
 
-1.	Open the `deployment.toml` file (stored in the `<MI_HOME>/conf` directory).
-2.	Add the following datasource configuration and update the values for your database.
+		- [Setting up a MySQL database](../../../setup/deployment/databases/setting-up-MySQL)
+		- [Setting up an MSSQL database](../../../setup/deployment/databases/setting-up-MSSQL)
+		- [Setting up an Oracle database](../../../setup/deployment/databases/setting-up-Oracle)
+		- [Setting up a Postgre database](../../../setup/deployment/databases/setting-up-PostgreSQL)
+		- [Setting up an IBM database](../../../setup/deployment/databases/setting-up-IBM-DB2)
 
-	```toml
+2.	Open the `deployment.toml` file (stored in the `<MI_HOME>/conf` directory).
+3.	Add the relevant datasource configuration and update the values for your database.
+
+	!!! Tip
+			If you are already using a JDBC user store (database) with another WSO2 product ([WSO2 API Manager](https://wso2.com/api-management/), [WSO2 Identity Server](https://wso2.com/identity-and-access-management/), or an instance of [WSO2 Enterprise Integrator 6.x.x](https://wso2.com/enterprise-integrator/6.6.0)), you can connect the same database to the Micro Integrator of WSO2 Enterprise Integrator 7 as explained below.
+
+	```toml tab='MySQL'
+	[[datasource]]
+	id = "WSO2_USER_DB"
+	url= "jdbc:mysql://localhost:3306/userdb"
+	username="root"
+	password="root"
+	driver="com.mysql.jdbc.Driver"
+	pool_options.maxActive=50
+	pool_options.maxWait = 60000
+	pool_options.testOnBorrow = true
+	```
+
+	```toml tab='MSSQL'
+	[[datasource]]
+	id = "WSO2_USER_DB"
+	url= "jdbc:sqlserver://<IP>:1433;databaseName=userdb;SendStringParametersAsUnicode=false"
+	username="root"
+	password="root"
+	driver="com.microsoft.sqlserver.jdbc.SQLServerDriver"
+	pool_options.maxActive=50
+	pool_options.maxWait = 60000
+	pool_options.testOnBorrow = true
+	```
+
+	```toml tab='Oracle'
 	[[datasource]]
-	id = "WSO2_CARBON_DB"
-	url= "jdbc:h2:./repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
-	username="username"
-	password="password"
-	driver="org.h2.Driver"
+	id = "WSO2_USER_DB"
+	url= "jdbc:oracle:thin:@SERVER_NAME:PORT/SID"
+	username="root"
+	password="root"
+	driver="oracle.jdbc.OracleDriver"
+	pool_options.maxActive=50
+	pool_options.maxWait = 60000
+	pool_options.testOnBorrow = true
+	```
+
+	```toml tab='PostgreSQL'
+	[[datasource]]
+	id = "WSO2_USER_DB"
+	url= "jdbc:postgresql://localhost:5432/userdb"
+	username="root"
+	password="root"
+	driver="org.postgresql.Driver"
+	pool_options.maxActive=50
+	pool_options.maxWait = 60000
+	pool_options.testOnBorrow = true
+	```
+
+	```toml tab='IBM DB'
+	[[datasource]]
+	id = "WSO2_USER_DB"
+	url="jdbc:db2://SERVER_NAME:PORT/userdb"
+	username="root"
+	password="root"
+	driver="com.ibm.db2.jcc.DB2Driver"
+	pool_options.maxActive=50
+	pool_options.maxWait = 60000
+	pool_options.testOnBorrow = true
 	```
```

Comment:
Are user store configurations missing? I couldn't locate.
```
[user_store]
class = "org.wso2.micro.integrator.security.user.core.jdbc.JDBCUserStoreManager"
type = "database"
read_only = false
```

Also we need to disable the file based user store for internal apis.
[internal_apis.file_user_store]
enable = false

Reply:
@sdkottegoda 
The [user_store] is given under point 4 below.
Isn't the [internal_apis.file_user_store] disabled when we enable an external user store? Do we need to purposely disable the file-based user store?

Reply:
fixed.